README.md

```diff
@@ -14,6 +14,8 @@ The Amazon ECS Container Agent is a component of Amazon Elastic Container Servic
 The best source of information on running this software is the
 [Amazon ECS documentation](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ECS_agent.html).
 
+Please note that from Agent version 1.20.0, Docker versions 1.9.0 or older are no longer supported.
```

Comment:
Please add a link to our docs page as well where information can be found and add a sentence here to the effect of "For more information, please visit the ECS agent documentation"